slsim/Sources/point_plus_extended_source.py

```diff
@@ -13,6 +13,7 @@ def __init__(
         kwargs_cut,
         variability_model=None,
         kwargs_variability_model=None,
+        list_type="astropy_table",
     ):
         """
```

Comment:
the class name does not indicate whether this is a population class or a single source object. Perhaps have a more uniform naming convention with plural